app/brave_dll.rc

```diff
@@ -0,0 +1,219 @@
+// Microsoft Visual C++ generated resource script.
+//
+#include "../../chrome/app/chrome_dll_resource.h"
+#include "../../chrome/app/chrome_command_ids.h"
+#include "brave_command_ids.h"
```

Comment:
I believe brave_command_ids.h is included in chrome_command_ids.h and only contains brave-specific ids so most of the ones listed here won't be there. This is one of the few cases where I think it's better to patch (vs large-scale copying)